package.json

```diff
@@ -16,12 +16,14 @@
     "access": "public"
   },
   "scripts": {
-    "build-tokens": "paragon build-tokens --source ./tokens/src --build-dir ./paragon/css --source-tokens-only",
+    "build-tokens": "paragon build-tokens --source ./tokens/src --build-dir ./paragon/css --source-tokens-only --output-references",
     "build-scss": "paragon build-scss --corePath ./paragon/core.scss --themesPath ./paragon/css/themes --defaultThemeVariants light",
     "build": "make build",
+    "build:watch": "nodemon --ignore dist -x \"make build\"",
     "paragon:help": "paragon help"
   },
   "devDependencies": {
-    "@openedx/paragon": "22.0.0-alpha.25"
+    "@openedx/paragon": "file:../paragon/openedx-paragon-20.18.1.tgz",
```

Comment:
[inform] Requires changes that are under review in Paragon `alpha`. Using local package for now via `npm pack`.